### PR TITLE
Hiv follow up

### PR DESCRIPTION
**Pull request by Michaelndula**
@moshonk I added ddl script for the hiv follow up

### CODE REVIEW

File: configuration/etl_ddl/hiv_care_enrolment/hiv_enrolment.sql [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    );
+
+    -- Log start time
+    INSERT INTO ssemr.etl_script_status(script_name, start_time) VALUES ('initial_creation_of_tables', NOW());
+    SET script_id = LAST_INSERT_ID();
+
+
+-- Create table hiv_care_enrolment --
+
+CREATE TABLE ssemr.ssemr_flat_encounter_hiv_care_enrolment
```
Comment (moshonk) [inline configuration/etl_ddl/hiv_care_enrolment/hiv_enrolment.sql:30]: @Michaelndula please retain the "create table" statement only

File: configuration/etl_ddl/hiv_care_follow_up/hiv_follow_up.sql [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        error       VARCHAR(255) DEFAULT NULL
+    );
+ 
+    INSERT INTO ssemr.etl_script_status(script_name, start_time) VALUES ('initial_creation_of_tables', NOW());
+    SET script_id = LAST_INSERT_ID();
+
+
+-- Create table hiv_care_follow_up --
+
+CREATE TABLE ssemr.ssemr_flat_encounter_hiv_care_follow_up
```
Comment (moshonk) [inline configuration/etl_ddl/hiv_care_follow_up/hiv_follow_up.sql:27]: @Michaelndula  please retain the "create table" statement only and remove DB name reference

Comment (Michaelndula) [reply configuration/etl_ddl/hiv_care_follow_up/hiv_follow_up.sql:27]: @moshonk well noted

Comment (Michaelndula) [reply configuration/etl_ddl/hiv_care_follow_up/hiv_follow_up.sql]: @moshonk I have removed the db name reference

File: configuration/etl_ddl/hiv_care_follow_up/hiv_follow_up.sql [hunk 3]
```diff
@@ -0,0 +1,75 @@
+DELIMITER //
```
Comment (moshonk) [inline configuration/etl_ddl/hiv_care_follow_up/hiv_follow_up.sql:1]: @Michaelndula please maintain this as a create table statement without the stored precedure

File: configuration/etl_ddl/hiv_care_enrolment/hiv_enrolment.sql [hunk 4]
```diff
@@ -0,0 +1,99 @@
+DELIMITER //
```
Comment (moshonk) [inline configuration/etl_ddl/hiv_care_enrolment/hiv_enrolment.sql:1]: @Michaelndula please maintain this as a create table statement without the stored precedure

**Pull request merged**